docs/content/posts/2024-10-30-release-kubernetes-operator-1.10.0.md

```diff
@@ -0,0 +1,76 @@
+---
+title:  "Apache Flink Kubernetes Operator 1.10.0 Release Announcement"
+date: "2024-10-30T18:00:00.000Z"
+authors:
+- mateczagany:
+  name: "Mate Czagany"
+aliases:
+- /news/2024/10/30/release-kubernetes-operator-1.10.0.html
+---
+
+The Apache Flink community is excited to announce the release of Flink Kubernetes Operator 1.10.0!
+
+The release includes several improvements to the autoscaler, and introduces a new Kubernetes custom resource called FlinkStateSnapshot to manage job snapshots.
+The process of job upgrades has also been enhanced which makes it possible to now use the last-state upgrade mode with session jobs.
+
+We encourage you to [download the release](https://flink.apache.org/downloads.html) and share your experience with the
+community through the Flink [mailing lists](https://flink.apache.org/community.html#mailing-lists) or
+[JIRA](https://issues.apache.org/jira/browse/flink)! We're looking forward to your feedback!
+
+## Highlights
+
+### FlinkStateSnapshot
+
+With this version comes also a new custom resource called FlinkStateSnapshot. 
+This is used to describe savepoint or checkpoint for a Flink job. 
+The savepoint/checkpoint fields found in FlinkDeployment and FlinkSessionJob status are therefore deprecated, and the operator will create new FlinkStateSnapshot resources for periodic, update and manual savepoints/checkpoints.
+
+Users can also create new FlinkStateSnapshot resources, which will instruct the operator to trigger new checkpoint/savepoint.
+
+This new feature is enabled by default, unless disabled by setting `kubernetes.operator.snapshot.resource.enabled` to false or if the FlinkStateSnapshot CRD was not found on the Kubernetes cluster.
+
+You can read more about this feature [here](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-release-1.10/docs/custom-resource/snapshots/).
+
+
+### Last-State Upgrade Mode
+
+For deployments using last-state upgrade mode, the operator - instead of forcefully deleting the JobManager pods to trigger a restart - will cancel the job via REST API and extract the last checkpoint info after cancellation if the job is healthy.
+This change makes it possible to finally use the last-state upgrade mode for session jobs as well.
+
+
+### Autoscaler Scale Down
+
+With the introduction of the configuration option `job.autoscaler.scale-down.interval`, the operator can now optimize multiple scale-down operations to a single one to prevent too many unnecessary downscales, thus improving job availability.
+Please note that `job.autoscaler.scale-up.grace-period` has been removed with this change.
+
+
+### Other Autoscaler Improvements
+- Autoscaling now takes into account the number of partitions in case of Kafka and Pulsar
```

Comment:
Assuming this improvement means is FLINK-36192. IIUC, Autoscaling already takes into account the number of partitions in case of Kafka and Pulsar in earlier version.

FLINK-36192 optimized the old logic when partitions or key groups cannot be evenly distributed to subtasks.

cc @huyuanfeng2018

Reply:
Thank you, you are right, I have made some changes to this line, please let me know if you find any other inaccuracies.